deployment/build-and-deploy-images.ps1

```diff
@@ -41,13 +41,30 @@ az acr build --image datagatewaymodule:$deploymentId --registry $acrName --file
 az acr build --image simulatedtemperaturesensormodule:$deploymentId --registry $acrName --file Distributed.IoT.Edge.SimulatedTemperatureSensorModule/Dockerfile .
 Set-Location -Path $deplymentDir
 
+# ----- Build and Push Containers (OPC Publisher)
+Write-Title("Build and Push Containers (OPC Publisher)")
+if (!(Test-Path .\..\..\Industrial-IoT-Temp))
+{
+    git clone -b feature/dapr-adapter https://github.com/simonjaeger/Industrial-IoT .\..\..\Industrial-IoT-Temp
```

Comment:
Wonder if you still wanted to continue building this from https://github.com/simonjaeger or https://github.com/suneetnangia?

Reply:
Agreed! This will break until that PR is merged however. But I am changing it.